ClassesHomework/Animal.cs

```diff
@@ -7,8 +7,41 @@
 namespace ClassesHomework
 {
     // write your Animal class here. 
-    //class Animal
-    //{
+    
+    public class Animal
+    {
+        public Animal()
+        {
+            
+        }
+        public bool Sit()
+        {
+            IsSitting = true;
+            return IsSitting;
+        }
+
+        public bool Stand()
+        {
+            IsSitting = false;
+            return IsSitting;
+        }
+
+        public int CelebrateBirthday()
+        {
+            //Age = ++Age;
+            this.Age = Age + 1;
+            return Age;
+        }
+        public virtual string Speak()
+        {
+            return "What am I?";
+        }
+        public virtual string Species { get; set; } = "unknown";
```

Comment:
There's nothing in the test that says Species can't be publicly settable, but is this the right access level for this property?